SpatialGDK/Source/SpatialGDK/Public/Utils/CheckoutRadiusConstraintUtils.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/SpatialClassInfoManager.h"
+#include "Schema/Interest.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogCheckoutRadiusConstraintUtils, Log, All);
+
+namespace SpatialGDK
+{
+class SPATIALGDK_API CheckoutRadiusConstraintUtils
+{
+public:
+	static QueryConstraint GetDefaultCheckoutRadiusConstraint();
+	static TMap<UClass*, float> GetActorTypeToRadius();
+	static TMap<float, TArray<UClass*>> DedupeDistancesAcrossActorTypes(const TMap<UClass*, float> ComponentSetToRadius);
```

Comment:
The tests only cover the `Dedupe` function. Is it intentional that the other functions are not covered?

Reply:
Yeah, I would have liked to add more tests, but during writing I realised I think a lot of the current functionality will change soon and the other functions I have refactored depend on somewhat arbitrary global state, so would be harder to write tests for now. Will write further tests in the future as I change other things.